plugins/gatsby-source-gathercontent/graphql/createItemsByTemplateName.js

```diff
@@ -0,0 +1,78 @@
+const camelcaseKeys = require('camelcase-keys');
+const { getAllTemplates, getStructure, getAllItems, getItemData } = require('../services');
+
+async function createItemsByTemplateName({
```

Comment:
This is from the original plugin code, I just moved it. But we're not using it atm.